libs/eo/landing-page/shell/src/lib/layout/eo-landing-page-column-layout.component.ts

```diff
@@ -0,0 +1,110 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  Component,
+  NgModule,
+  ViewEncapsulation,
+  ChangeDetectionStrategy,
+  Input,
+} from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+export enum layoutTypeEnum {
+  FULL = 'full',
+  SMALL_FIRST = 'smallFirst',
+  LARGE_FIRST = 'largeFirst',
+}
```

Comment:
issue: Use this enum naming style and enum key naming style:
```suggestion
export enum LayoutType {
  Full = 'full',
  SmallFirst = 'smallFirst',
  LargeFirst = 'largeFirst',
}
```

Reply:
suggestion: Consider a union type instead of an enum, this makes it easier to use in component templates and is recommended by experts.
```suggestion
export type LayoutType = 'full' | 'smallFirst' | 'largeFirst';
```

Reply:
It is! Updated :-)